lib/brewdler/dsl.rb

```diff
@@ -1,7 +1,8 @@
 module Brewdler
   class Dsl
-    def initialize(input)
+    def initialize(input, mode)
```

Comment:
Might want to use a `options={}` hash here instead and pass `:mode => :install` to make it a bit more Ruby-like (and handle the case where we add more args later)